db/data/20200615144002_add_implementing_organisations_to_programmes.rb

```diff
@@ -0,0 +1,23 @@
+class AddImplementingOrganisationsToProgrammes < ActiveRecord::Migration[6.0]
+  def up
+    programmes = Activity.programme
+    programmes.each do |programme|
+      next if programme.implementing_organisations.present?
+      next unless programme.extending_organisation_id
+      extending_organisation = Organisation.find(programme.extending_organisation_id)
+      implementing_organisation = ImplementingOrganisation.new(name: extending_organisation.name,
+                                                               organisation_type: extending_organisation.organisation_type,
+                                                               reference: extending_organisation.iati_reference,
+                                                               activity_id: programme.id)
+      if implementing_organisation.valid?
```

Comment:
It feels the same here, with no implementing org the record will not pass IATI validation, so perhaps `save!` ?

Reply:
@lozette what is the thinking around not raising an exception during the ingest?

Reply:
No you're right, we should raise an exception!